tools/ci_connector_ops/ci_connector_ops/pipelines/actions/environments.py

```diff
@@ -408,13 +408,24 @@ def with_gradle(
         .with_env_variable("VERSION", "23.0.1")
         .with_exec(["sh", "-c", "curl -fsSL https://get.docker.com | sh"])
         .with_exec(["mkdir", "/root/.gradle"])
-        .with_mounted_cache("/root/.gradle", root_gradle_cache, sharing=CacheSharingMode.LOCKED)
-        .with_exec(["rm", "-f", "/root/.gradle/caches/journal-1/journal-1.lock"])
         .with_exec(["mkdir", "/airbyte"])
         .with_mounted_directory("/airbyte", context.get_repo_dir(".", include=include))
         .with_mounted_cache("/airbyte/.gradle", airbyte_gradle_cache, sharing=CacheSharingMode.LOCKED)
         .with_workdir("/airbyte")
     )
+    if context.is_ci:
```

Comment:
@alafanechere Can we leave a comment around this that explains
1. What we are doing
2. Why
3. When we should remove it and whos responsible